en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md

```diff
@@ -86,6 +86,57 @@ WSO2 API Controller, **apictl** allows to create and deploy APIs without using W
                         `--oas` : Provide an OpenAPI specification file/URL for the API   
                         `--force` or `-f` : To overwrite directory and create project 
 
+        !!! note
+            You can define scopes for a resource when defining a Swagger2 or OpenAPI3 specification to generate an API.
+
+            !!! example
+                ```yaml
+                openapi: 3.0.0
+                info:
+                title: Online-Store
+                version: v1.0.0
+                description: This API contains operations related to online shopping store.
+                x-wso2-basePath: /store/{version}
+                x-wso2-production-endpoints:
+                urls:
+                    - http://products
+                paths:
+                /products:
+                    get:
+                    responses:
+                        "200":
+                        description: successful operation
+                /products/{productId}:
+                    get:
+                    parameters:
+                        - name: productId
+                        in: path
+                        required: true
+                        schema:
+                            type: string
+                    security: 
+                        - 
+                        default: 
+                            - "products:read"
+                    responses:
+                        "200":
+                        description: successful operation
+                components: 
+                securitySchemes: 
+                    default: 
+                    type: "oauth2"
+                    flows: 
+                        implicit: 
+                        authorizationUrl: "https://test.com"
+                        scopes: 
+                            products:read: ""
+                        x-scopes-bindings: 
+                            products:read: "admin"
+                ```
+
+            You need to first define the scope name (products:read) under `security > default` section inside the required resource and then define the role biniding under the `securitySchemes` section.
```

Comment:
Shall we change the small typo "biniding" to "binding" here?

Reply:
done